src/main/typescript/wcardinal/ui/d-slider-value.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2019 Toshiba Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DBaseState } from "./d-base-state";
+import { DTextBase, DTextBaseOptions, DThemeTextBase } from "./d-text-base";
+
+export interface DSliderValueOptions<
+	VALUE = number,
+	THEME extends DThemeSliderValue = DThemeSliderValue
+> extends DTextBaseOptions<VALUE, THEME> {
+	value?: number;
+	precision?: number;
+	round?( value: number ): number;
+}
+
+export interface DThemeSliderValue extends DThemeTextBase {
+	getPrecision(): number;
+}
+
+const toPrecision = <VALUE, THEME extends DThemeSliderValue>(
+	theme: DThemeSliderValue,
+	options: DSliderValueOptions<VALUE, THEME> | undefined
+): number => {
+	if( options && options.precision ) {
+		return options.precision;
+	}
+	return theme.getPrecision();
+};
```

Comment:
Please refer to the comment on the `DSliderValue#toRound` for the reason why this function is not required.

```suggestion
```